drivers/pinmux/stm32/pinmux_stm32f4.h

```diff
@@ -418,6 +418,8 @@
 #define STM32F4_PINMUX_FUNC_PG9_USART6_RX   \
 	(STM32_PINMUX_ALT_FUNC_8 | STM32_PUSHPULL_NOPULL)
 
+#define STM32F4_PINMUX_FUNC_PG8_USART6_RTS \
```

Comment:
Are these STM32F479X specific?
If yes, they should be under appropriate flags, else rewrite the commit message/title

Reply:
Not really, reworded.